docs/csharp/quick-starts/numbers-in-csharp-local.md

```diff
@@ -0,0 +1,373 @@
+
+You can force a different order of operation by adding parentheses around
+the operation or operations you want performed first. Add the following
+lines and run again:
+
+```csharp
+d = (a  + b) * c;
+Console.WriteLine(d);
+```
+
+Explore more by combining many different operations. Add something like
+the following lines at the bottom of your `Main` method. Try `dotnet run` again.
+
+```csharp
+d = (a + b) - 6 * c + (12 * 4) / 3 + 12;
+Console.WriteLine(d);
+```
+
+You may have noticed an interesting behavior for integers. Integer
+division always produces an integer result, even when the mathematical
+result would include a decimal or fraction portion.
+
+If you haven't seen this behavior, try the following code at the end
+of your `Main` method:
+
+```csharp
+int e = 7;
+int f = 4;
+int g = 3;
+int h = (e  + f) / g;
+Console.WriteLine(h);
+```
+
+Type `dotnet run` again to see the results.
+
+Before moving on, let's take all the code you've written in this
+section and put it in a new method. Call that new method `OrderPrecedence`.
+You should end up with something like this:
+
+```csharp
+using System;
+
+namespace NumbersInCSharp
+{
+    class Program
+    {
+        static void WorkingWithIntegers()
+        {
+            int a = 18;
+            int b = 6;
+            int c = a + b;
+            Console.WriteLine(c);
+            c = a - b;
+            Console.WriteLine(c);
+            c = a * b;
+            Console.WriteLine(c);
+            c = a / b;
+            Console.WriteLine(c);
+        }
+
+        static void OrderPrecedence()
+        {   
+            int a = 5;
+            int b = 4;
+            int c = 2;
+            int d = a + b * c;
+            Console.WriteLine(d);
+
+            d = (a  + b) * c;
+            Console.WriteLine(d);
+
+            d = (a + b) - 6 * c + (12 * 4) / 3 + 12;
+            Console.WriteLine(d);
+
+            int e = 7;
+            int f = 4;
+            int g = 3;
+            int h = (e  + f) / g;
+            Console.WriteLine(h);
+        }
+
+        static void Main(string[] args)
+        {
+            WorkingWithIntegers();
+
+            OrderPrecedence();
+
+        }
+    }
+}
+```
+
+## Explore integer precision and limits
+That last sample showed you that integer division truncates the result.
+You can get the **remainder** by using the **modulo** operator, the `%` character. Try the following code in your `Main` method:
+
+```csharp
+int a = 7;
+int b = 4;
+int c = 3;
+int d = (a  + b) / c;
+int e = (a + b) % c;
+Console.WriteLine($"quotient: {d}");
+Console.WriteLine($"remainder: {e}");
+```
+
+The C# integer type differs from mathematical integers in one other
+way: the `int` type has minimum and maximum limits. Add this code to
+your `Main` method to see those limits:
+    
+```csharp
+int max = int.MaxValue;
+int min = int.MinValue;
+Console.WriteLine($"The range of integers is {min} to {max}");
```

Comment:
This produces numbers that are hard to parse. Decimal separators help: {min:n0} and {max:n0}

Reply:
I left that out on purpose, because I haven't explained format strings yet. But I agree with your comment.